lib/identity_cache/configuration_dsl.rb

```diff
@@ -70,21 +72,22 @@ def cache_has_many(association, embed: :ids, inverse_name: nil)
       # == Example:
       #   class Product
       #     cache_has_one :store, embed: true
-      #     cache_has_one :vendor
+      #     cache_has_one :vendor, embed: :id
       #   end
       #
       # == Parameters
       # +association+ Symbol with the name of the association being cached
       #
       # == Options
       #
-      # * embed: Only true is supported, which is also the default, so
-      #   IdentityCache will keep the values for this association in the same
-      #   cache entry as the parent, instead of its own.
+      # * embed: If `true`, IdentityCache will embed the associated record
+      #   in the cache entries for this model, as well as all the embedded
+      #   associations for the associated record recursively.
+      #   If `:id`, it will only embed the id for the associated record.
       # * inverse_name: The name of the parent in the association ( only
       #   necessary if the name is not the lowercase pluralization of the
       #   parent object's class)
-      def cache_has_one(association, embed: true, inverse_name: nil)
+      def cache_has_one(association, embed:, inverse_name: nil)
```

Comment:
This bugs me because it isn't consistent, but I'm fine with fixing that in another release as you mention. 👍